Applications/PlazaDetails/ServiceMaturity.qmd

```diff
@@ -4,34 +4,35 @@ aliases:
+Each service in the openEO algorithm plaza is assigned a maturity level that indicates its qualitative documentation and performance. Currently, five different maturity levels for each service are available. The prototype is the primary and default level, followed by incubating, verifying, validating, and operating as advanced services. These levels are determined solely based on software readiness and user documentation criteria. These criteria are generally designed to ensure that the service meets specific standards and provides customers with a certain level of quality.
+
 
 
 ## Level 1: Prototype
 
-By default, every published service will have a prototype level. It is expected that users consider the following points when publishing a service:
+By default, every published service has a prototype level. It is expected that users consider the following points when publishing a service:
 
 - The service is executable, and basic logging information is supported.
 
 - A possible reference or a general overview of what it tries to implement is provided as service metadata.
 
-:::{callout-note}
+::: {.callout-note}
 
-If your service satisfies the criteria for a higher level, you can request an upgrade anytime once your service is published. Nevertheless, please note that every criterion must be satisfied for an upgrade.
+Suppose a service meets the criteria for a higher level. In that case, an upgrade can be requested anytime after publication. It's essential to ensure that all requirements are satisfied before the upgrade can be approved.
 
 :::
 
 ## Level 2: Incubating
 
-In addition to the criteria for prototype level, a few additional criteria, as mentioned below, need to be satisfied to be upgraded to incubating service.
+In addition to the criteria for prototype level, a service needs to meet the following requirements to be an incubating service:
 
-- Service metadata should also include an example of executing the service along with the expected output format.
+- Service metadata should include an example along with the expected output format.
   
 - An approximate assumption on how much user credit is required to execute a service should be provided.
 
-:::{callout-note}
+::: {.callout-note}
 
-Note that no added value will be associated with services with a prototype or incubating levels. In other words, approximate credit will include added value cost only on services that are either verified, validated or operational.
+Note that no added value is associated with services in a prototype or incubating levels. In other words, added value costs are included in services that are either verified, validated or operational.
```

Comment:
I believe it is 'on a level'.
Preferably even use in a prototype or incubating stage